From 9cf72900b67718dcd1d258d2fcc325f68c8efd69 Mon Sep 17 00:00:00 2001
From: Julien Castelain <julien.castelain@liferay.com>
Date: Wed, 25 Mar 2020 12:58:15 +0100
Subject: [PATCH] LPS-85326 Remove check for Webkit browsers

As described in this issue:

https://issues.liferay.com/browse/LPS-85326

Converting links in an unordered list (in reverse order) throws an error
in non webkit browsers.

This patch must have been skipped when we updated CKEditor here:

https://github.com/liferay/liferay-ckeditor/commit/328017c65063ac18119e244fec92d35a4c2fedef
---
 core/selection.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/selection.js b/core/selection.js
index 72a02ef9f0..4da7aff707 100644
--- a/core/selection.js
+++ b/core/selection.js
@@ -2278,7 +2278,7 @@
 
 					var nativeRange = this.document.$.createRange();
 
-					if ( range.collapsed && CKEDITOR.env.webkit && rangeRequiresFix( range ) ) {
+					if ( range.collapsed && rangeRequiresFix( range ) ) {
 						// Append a zero-width space so WebKit will not try to
 						// move the selection by itself (https://dev.ckeditor.com/ticket/1272).
 						var fillingChar = createFillingCharSequenceNode( this.root );
